spec.html

```diff
@@ -39622,39 +39622,45 @@ <h1>%RegExpStringIteratorPrototype% [ %Symbol.toStringTag% ]</h1>
           <p>The initial value of the %Symbol.toStringTag% property is the String value *"RegExp String Iterator"*.</p>
           <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
         </emu-clause>
+      </emu-clause>
```

Comment:
Is there anything we can do with tooling to catch these kinds of misindented clauses?

Reply:
ecmaspeak-py has code that checks whether the properties of an object are listed in a consistent order (basically alphabetical), and when those properties have a sibling section that isn't a property, the code complains.